src/utils/ADempiere/evaluator.js

```diff
@@ -14,27 +14,31 @@
 class evaluator {
   /**
    * Evaluate logic's
-   * @param {string} parentUuid Parent (Window / Process / Smart Browser)
+   * @param {string} parentUuid Parent (Window)
+   * @param {string} containerUuid Parent (Tab / Process / Smart Browser)
```

Comment:
This change does not have a change functionality, please remove it from changeset

Reply:
@yamelsenih Although the update of the comment did not affect any functionality, the change was necessary to document the update of the parameters, for those who develop and use the 'evaluator', however the change of the comment was removed on request.